doc/REPLICATED.md

```diff
@@ -0,0 +1,79 @@
+# Installing Waffle Takeout
+
+Please submit a request on [https://takeout.waffle.io](https://takeout.waffle.io) for more information on pricing and to register for a 45 day trial.
+
+Getting Started guides and other Waffle.io documentation can be found at [here](https://github.com/waffleio/waffle.io/wiki).
+
+## Prerequisites
+
+* A dedicated host with Ubuntu 14.04 or greater installed
+* A Waffle Takeout license file
+
+## Host Machine
+
+The standard setup consists of one host, running the following services:
+
+* Migrations
+* Poxa
+* Rally
+* Hedwig
+* Hooks
+* API
+* App
+* Nginx
+
+If you are using EC2 we recommend the m4.large instance types.
+
+For other setups we recommend hosts with 8 gigs of RAM and 2 CPUs.
+
+## Register a GitHub OAuth app
+
+Waffle Takeout talks to GitHub Enterprise and GitHub.com via OAuth. You will need to create an OAuth app on your GitHub Enterprise installation or GitHub.com that Waffle Takeout can connect to.
+
+The OAuth app registered will use the domain name pointing to your Platform host for the Homepage URL (e.g. https://waffle.company.com).
```

Comment:
instead of `the domain name pointing to your Platform host`, let's say `the network routable address to your host machine`